bindings/nodejs/types/network.ts

```diff
@@ -9,8 +9,7 @@ export enum Network {
 /** Basic Auth or JWT */
 export type Auth = {
     jwt?: string;
-    username?: string;
-    password?: string;
+    basicAuthNamePwd?: [string, string];
```

Comment:
Maybe also in the .change as this is breaking

Reply:
This was already broken :trollface: But will add